### DISCUSSION TITLE
Funcs C

### DISCUSSION THREAD

**Discussion by 11django11** [11django11/test-project#11] [Q&A] [answered]
How to write C funcs?

**Comment by seldish-og** [comment, marked as answer]
Example

Given below is the source code for a function called max(). This function takes two parameters num1 and num2 and returns the maximum value between the two −

/* function returning the max between two numbers */
int max(int num1, int num2) {

   /* local variable declaration */
   int result;
 
   if (num1 > num2)
      result = num1;
   else
      result = num2;
 
   return result; 
}
